src/components/Avatar/__stories__/utils/getClosestNumber.ts

```diff
@@ -1,4 +1,4 @@
-export function getClosestNumber(targetNumber: number, availableNumbers: number[]) {
+export const getClosestNumber = (targetNumber: number, availableNumbers: number[]) => {
```

Comment:
What's the intention of the change?

Reply:
Nothing special. Just a little optimization